mfr/extensions/codepygments/render.py

```diff
@@ -25,6 +27,9 @@ def __init__(self, *args, **kwargs):
         self.metrics.add('pygments_version', pygments.__version__)
 
     def render(self):
+        file_size = os.path.getsize(self.file_path)
+        if file_size > settings.MAX_SIZE:
+            raise(cp_exceptions.FileToLargeException('Text files larger than 64kb are not rendered; please download the file to view.'))
```

Comment:
My only concern with this is that if we decide to change `MAX_SIZE`, this figure won't be accurate. 

Not blocking, but consider using some util to format `MAX_SIZE`. I don't know of any built-ins that do this, but something like this would work:

```
In [1]: %paste
def stringify_bytecount(b):
    i = 0
    sz = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
    while b >= 1024 and i < len(sz)-1:
        i += 1
        b /= 1024.0
    return '{} {}'.format(round(b, 1), sz[i]).replace('.0 ', ' ')

## -- End pasted text --

In [2]: stringify_bytecount(6553300000000000000000000000)
Out[2]: '5420.8 YB'

In [3]: stringify_bytecount(65536)
Out[3]: '64 KB'
```